google/cloud/bigquery/enums.py

```diff
@@ -256,28 +221,6 @@ class SqlTypeNames(str, enum.Enum):
     DATETIME = "DATETIME"
 
 
-class SqlParameterScalarTypes:
```

Comment:
Indeed, it was causing too many problems, failing tests, etc., and moving it out of the enums was the most efficient solution. As a rule of thumb, since enums are (in)directly imported almost everywhere, enums themselves should not import much.